src/App.js

```diff
@@ -9,15 +9,16 @@ import React from "react";
 const { setConfig, getValue } = Config;
 
 const testApiRoot =
-  "https://structuredcontentdev.bcg.ad.bcgov.us/api/hub/news/news";
-const prodApiRoot = "https://services.baltimorecountymd.gov/api/hub/news/news";
+  "https://structuredcontentdev.bcg.ad.bcgov.us/api/hub/structuredContent/news";
```

Comment:
you should be using `testservices`, not `structuredcontentdev`

Reply:
done. I was not able to test . it is blowing up on a hubcontroller, which i mentioned this morning. Even the pet which was working after app pool recycled is not working

Reply:
`testservices.bcg.ad.bcgov.us` should be `testservices.baltimorecountymd.gov`. I suspect that will give you different results.